vars/setupAndRunScaleTest.groovy

```diff
@@ -22,8 +22,7 @@ def call(){
                 /usr/bin/time  -o ~/time_taken_namespaces -f  'real-%E user-%U system-%S' ./namespaces.sh;
                 ./clusterbuster --cleanup;
                 sleep 60;
-                oc label node worker-0 node-role.kubernetes.io/infra="";
-                oc label node worker-1 node-role.kubernetes.io/infra="";
+                oc label node $(oc get node --selector='node-role.kubernetes.io/worker' --no-headers | grep 'worker-1\|worker-2' | awk '{print $1}') node-role.kubernetes.io/infra="";
```

Comment:
Please change 'worker-1\|worker-2' to 'worker-1\\|worker-2' since jenkins required double slash instead of  single slash